src/services/track-field/tf-season/tf-season.schema.ts

```diff
@@ -0,0 +1,64 @@
+// // For more information about this file see https://dove.feathersjs.com/guides/cli/service.schemas.html
+import { resolve } from '@feathersjs/schema'
+import { Type, getValidator, querySyntax } from '@feathersjs/typebox'
+import type { Static } from '@feathersjs/typebox'
+
+import type { HookContext } from '../../../declarations'
+import { dataValidator, queryValidator } from '../../../validators'
+import { toLowerCaseProperty } from '../../../utilities/property-name-converter';
+
+
+// Main data model schema
+
+export const seasonSchema = Type.Object(
+    {
+      id: Type.Number(),
+      name: Type.String(),
+      date: Type.String({ format: 'date' }),
```

Comment:
could we take this as a followup:
we should only need/care YYYYMMDD from the `date`